### DISCUSSION TITLE
Xiaomi 11 cell phone full screen bad adaptation

### DISCUSSION THREAD

**Discussion by Dobulinnac** [koreader/koreader#10409] [General]
When I select full-screen auto-match, the progress bar below cannot be displayed when reading in vertical screen, and it appears only when rotated to horizontal screen. If you choose to hide the bangs in the app, the progress bar can be displayed, but a blank area will be left underneath.
![Screenshot_2023-05-10-05-16-57-403_org.koreader.launcher.fdroid.jpg](https://github.com/koreader/koreader/assets/126112021/35c68ef3-957d-4b75-ab5b-c4c0c3c91aef)

![IMG_20230510_052453.jpg](https://github.com/koreader/koreader/assets/126112021/b8543fcb-1c28-41b1-b14d-1787d46d830a)

**Comment by Frenzie** [comment]
Could be the same as <https://github.com/koreader/koreader/issues/9446>?

**Reply by Dobulinnac** [reply]
It may be a problem with the full screen display, the previous system had a full screen display setting, the new system does not have this setting. So many apps have a similar problem, in that either a piece is cut off above or a piece is cut off below. Thank you for making such a wonderful app!

**Reply by Frenzie** [reply]
The program supports rounded corners. It's only dealing with camera cutouts that can be a bit more awkward.

**Reply by Dobulinnac** [reply]
V2023.06 this problem was solved! Thanks very much!
